libs/core/build.gradle

```diff
@@ -44,13 +44,13 @@ if (!isEclipse) {
   }
 
   compileJava9Java {
```

Comment:
This should be `compileJava11Java`

Reply:
We also need to change the source sets above, etc.

Here's what I suggest, let's make changing this to Java 11 a separate change, so it can be reviewed separately. Can you open another pull request for this? Then we can merge that one, and rebase this PR on top of that.

Reply:
@jasontedor, I've rebased this PR now that the separate PR to update the MRJARs to Java11 has been merged.